cl/lib/timezone_helpers.py

```diff
@@ -0,0 +1,38 @@
+from datetime import date, datetime, time
+
+import pytz
+
+from cl.recap.constants import COURT_TIMEZONES
+
+
+def convert_to_court_timezone(
+    datetime_filed: datetime, court_id: str
+) -> datetime:
+    """Convert a docket entry datetime filed to the court timezone it belongs
+    to.
+
+    :param datetime_filed: The docket entry datetime filed
+    :param court_id: The court id to which docket entries belong, used for
+    timezone conversion.
+    :return: A date object in the court timezone.
+    """
+    court_timezone = pytz.timezone(COURT_TIMEZONES.get(court_id, "US/Eastern"))
+    return datetime_filed.astimezone(court_timezone)
+
+
+def localize_date_and_time(
+    court_id: str, date_filed: date | datetime
+) -> tuple[date, time | None]:
```

Comment:
This is a pretty small thing, but `convert_to_court_timezone` orders the args as date-court, but localize_date_and_time orders them as court-date, which is a bit confusing if you want to memorize it. 

Can we choose one or the other?

Reply:
Sure, I changed the order to court-date.